policyengine_us/variables/gov/states/hi/tax/income/exemptions/hi_exemptions.py

```diff
@@ -0,0 +1,44 @@
+from policyengine_us.model_api import *
+
+
+class hi_exemptions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Hawaii exemptions"
+    unit = USD
+    documentation = (
+        "https://files.hawaii.gov/tax/forms/2022/n11ins.pdf#page=20"
+    )
+    definition_period = YEAR
+    defined_for = StateCode.HI
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.hi.tax.income.exemptions
+        person = tax_unit.members
+
+        num_exemp = tax_unit("exemptions", period)
```

Comment:
we can use `head_is_disabled` and `spouse_is_disabled` as inputs

Reply:
I've changed the inputs of the variable. However, I found since the function "exemptions" will return the total number of exemptions in a tax unit, say there's a head and a spouse, then when we calculate the exemptions for the head in the python file (if you could see the latest version of the variable file), how do we know which parts of the total exemptions belong to the head and which parts belong to the spouse?